scripts/algorithms/l2-svm-predict.dml

```diff
@@ -25,6 +25,19 @@
 # Given ground truth labels, the script will compute an 
 # accuracy (%) for the predictions
 #
+# INPUT PARAMETERS:
+# ---------------------------------------------------------------------------------------------
+# NAME      TYPE        DEFAULT     MEANING
+# ---------------------------------------------------------------------------------------------
+# X         String      ---         Location to read the test feature matrix X
+# Y         String      ---         Location to read the true label matrix Y
+# confusion String      "csv"       Location to write confusion matrix
```

Comment:
There is no default here.  Also, can you note that this is optional, and only valid if `Y` is supplied?